vsphere/telegraf.conf

```diff
@@ -0,0 +1,198 @@
+ [[outputs.influxdb_v2]]
+  ## The URLs of the InfluxDB cluster nodes.
+  ##
+  ## Multiple URLs can be specified for a single cluster, only ONE of the
+  ## urls will be written to each interval.
+  ## urls exp: http://127.0.0.1:9999
+  urls = ["http://localhost:9999"]
```

Comment:
Make an environment variable

Reply:
I comment on the login and hosts, no only in this one, in all. But make an environment variable the name of a host is too much. I comment on the line, so, the user can see what needs to edit.